src/api.js

```diff
@@ -65,14 +94,13 @@ module.exports = class {
     if (MyQDeviceId) return Promise.resolve(MyQDeviceId);
 
     return this.getDeviceList(options).then(devices => {
-      const withoutGateways = _.reject(devices, {MyQDeviceTypeId: GATEWAY_ID});
-      const ids = _.map(withoutGateways, 'MyQDeviceId');
-      const {0: MyQDeviceId, length} = ids;
-      if (length === 0) throw new Error('No controllable devices found');
-
-      if (length === 1) {
-        this.options = _.extend({}, this.options, {MyQDeviceId});
-        return MyQDeviceId;
+      const withoutGateways = _.reject(devices, {device_type: 'hub'});
```

Comment:
I have ethernetgateway as a device type causing the multiple devices error to be thrown at the end of this function

Reply:
I've adjusted to       

const withoutHubs = _.reject(devices, {device_type: 'hub'});
const withoutGateways = _.reject(withoutHubs, {device_type: 'ethernetgateway'});